### PR TITLE
Update authorization.md to include mutation#authorized?

### PR DESCRIPTION
**Pull request by fwolfst**
Mention both  `#authorized?` methods of Mutations. The instance method lacks explanation, but at least its mentioned now.

### CODE REVIEW

**Comment by rmosolgo** [comment]
Hey, this is described in the mutation authorization guide: https://graphql-ruby.org/mutations/mutation_authorization.html#can-this-user-perform-this-action

How about linking to that guide instead? (Or improving that guide?)

**Comment by fwolfst** [comment]
> How about linking to that guide instead? (Or improving that guide?)

That makes sense, I will update the PR.

**Comment by fwolfst** [comment]
Updated the PR. So, mutation has `ready?`, but also `self.authorized?`. Does it make sense to mention the latter (second commit)? Iiuc, it is executed before any of the other mentioned methods.

File: guides/authorization/authorization.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Schema members have `authorized?` methods which will be called during execution:
 - Type classes have `.authorized?(object, context)` class methods
 - Fields have `#authorized?(object, args, context)` instance methods
 - Arguments have `#authorized?(object, arg_value, context)` instance methods
+- Mutations have `#authorized?(object, context)` class method and a `#authorized?(hash)` instance method
```
Comment (rmosolgo) [inline guides/authorization/authorization.md:19]:
```suggestion
- Mutations and Resolvers have `.authorized?(object, context)` class methods and `#authorized?(args)` instance methods
```

**Comment by rmosolgo** [comment]
Thanks for this improvement -- sorry it took me so long to get back to it!

**Pull request merged**